File: src/main/java/com/example/ssauc/user/cash/dto/ChargeDto.java
package com.example.ssauc.user.cash.dto;

import java.time.LocalDateTime;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ChargeDto {
    private Long chargeId;
    private String chargeType;
    private Long amount;
    private String status;
    private LocalDateTime createdAt;
    private String receiptUrl;
}

File: src/main/java/com/example/ssauc/user/cash/dto/ChargeDto.kt
package com.example.ssauc.user.cash.dto

import lombok.*
import java.time.LocalDateTime

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
class ChargeDto {
    private val chargeId: Long? = null
    private val chargeType: String? = null
    private val amount: Long? = null
    private val status: String? = null
    private val createdAt: LocalDateTime? = null
    private val receiptUrl: String? = null
}
